folly/json_pointer.cpp

```diff
@@ -49,7 +49,7 @@ Expected<json_pointer, json_pointer::parse_error> json_pointer::try_parse(
 json_pointer json_pointer::parse(StringPiece const str) {
   auto res = try_parse(str);
   if (res.hasValue()) {
-    return std::move(res.value());
+    return res.value();
```

Comment:
Presumably we want to avoid copying `vector<string>`.

Reply:
Is this from clang-tidy or is this manual?

Reply:
clang-tidy. Maybe because of the earlier std::move.

Reply:
This seems wrong though. `res.value()` is an lvalue-ref and so this code without `std::move` logically invokes the copy-constructor of `json_pointer`, which contains a `vector<string>`. The goal of `std::move` is to invoke the move-constructor instead.

Reply:
Reverted since this is cosmetic anyway.